BlogTemplate/Models/BlogDataStore.cs

```diff
@@ -212,6 +212,7 @@ public void AppendPostInfo(Post post, XElement rootNode)
             rootNode.Add(new XElement("LastModified", post.LastModified.ToString("o")));
             rootNode.Add(new XElement("IsPublic", post.IsPublic.ToString()));
             rootNode.Add(new XElement("Excerpt", post.Excerpt));
+            rootNode.Add(new XElement("EnableComments", post.EnableComments));

```

Comment:
Do you have to convert this to a string first? Like post.EnableComments.ToString()? That is what we were doing for IsPublic, but maybe it's not necessary.